Source/FullScreenPokemon.ts

```diff
@@ -1013,14 +1013,20 @@ module FullScreenPokemon {
          * @todo Extend the use for any registered item, not just the bicycle.
          */
         keyDownSelect(thing: ICharacter, event?: Event): void {
+            var item: string = thing.FSP.ItemsHolder.getItem("SelectItem");
+
+            if (!item) {
+                thing.FSP.displayMessage(thing, "You can register an item to the SELECT key");
```

Comment:
Just return for now. Whether this should be displayed is out of scope for this PR.